spec/unit/lib/cloud_controller/paging/sequel_paginator_spec.rb

```diff
@@ -8,6 +8,13 @@ module VCAP::CloudController
 
     describe '#get_page' do
       let(:dataset) { AppModel.dataset }
+      table_name = SequelPaginator::EXCLUDE_FROM_PAGINATION_WITH_WINDOW_FUNCTION[0]
```

Comment:
Local variables within a context/describe block are uncommon. I would simply use the hard-coded model (i.e. `VCAP::CloudController::Event` instead and move the `let`s into an own context.

Reply:
Ok, thank you